modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -4,36 +4,42 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation, jre
 
-Recommended minimum hardware is:
+If you plan to install Mule and run it on premises, review these minimum hardware and software requirements before you install.
+
+== Minimum Hardware Requirements
 
 * 2GHz CPU or 1 virtual CPU in virtualized environments
-* 1GB of RAM
+* 1GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
```

Comment:
This addition is technically incorrect and misleading. Leave this as it was.

Reply:
Updated.